supertokens_python/recipe/passwordless/smsdelivery/service/backward_compatibility/__init__.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import json
+from os import environ
+from typing import Any, Awaitable, Callable, Dict, Union
+
+from httpx import AsyncClient
+from supertokens_python.ingredients.smsdelivery.service.supertokens import \
+    SUPERTOKENS_SMS_SERVICE_URL
+from supertokens_python.ingredients.smsdelivery.types import \
+    SMSDeliveryInterface
+from supertokens_python.logger import log_debug_message
+from supertokens_python.supertokens import AppInfo
+
+from ....types import TypePasswordlessSmsDeliveryInput
+
+
+class InvalidSendCustomSmsResponse(Exception):
+    pass
+
+
+def default_create_and_send_custom_sms(app_info: AppInfo):
+    async def func(sms_input: TypePasswordlessSmsDeliveryInput, _user_context: Dict[str, Any]):
+        if ('SUPERTOKENS_ENV' in environ) and (environ['SUPERTOKENS_ENV'] == 'testing'):
+            return
+        sms_input_json = {
+            'appName': app_info.app_name,
+            'type': 'PASSWORDLESS_LOGIN',
+            'phoneNumber': sms_input.phone_number,
+            'codeLifetime': sms_input.code_life_time,
+        }
+        if sms_input.user_input_code:
+            sms_input_json['userInputCode'] = sms_input.user_input_code
+        if sms_input.url_with_link_code:
+            sms_input_json['urlWithLinkCode'] = sms_input.url_with_link_code
+
+        try:
+            async with AsyncClient() as client:
+                res = await client.post(  # type: ignore
+                    SUPERTOKENS_SMS_SERVICE_URL,
+                    json={
+                        "smsInput": sms_input_json,
+                    },
+                    headers={'api-version': '0'}
+                )
+                # TODO: Do we need an equivalent of "err.response === undefined"?
+                if res.status_code == 429:
+                    raise InvalidSendCustomSmsResponse(res.text)
+                return
+        except InvalidSendCustomSmsResponse as e:
+            raise e
```

Comment:
If you are raising here, how will the print statement below get printed out?

Reply:
Fixed.